openedge.properties

```diff
@@ -12,3 +12,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
 2.14.4.date=2021-11-23
```

Comment:
Only one version can be compatible with `LATEST` at a time. You need to make this range end with 9.8

Reply:
Done